<launch >
  <arg name="fps" default="5" />
  <node pkg="debug_pkg" namespace="camera_pkg" exec="inference_comparison_node" output="log">
      <param name="resize_images" type="bool" value="True" /> 
      <param name="display_topic_enable" type="bool" value="False" /> 
      <param name="fps" type="int" value="$(var fps)" />
      <param name="input_dir" type="str" value="/workspaces/deepracer-devel/output/test/" />
  </node>
  <node pkg="sensor_fusion_pkg" namespace="sensor_fusion_pkg" exec="sensor_fusion_node" output="log">
      <param name="image_transport" type="str" value="compressed" />
  </node>
  <node pkg="inference_pkg" namespace="inference_pkg_tflite" exec="inference_node" output="log">
    <param name="inference_engine" type="str" value="TFLITE" />
  </node>
  <node pkg="inference_pkg" namespace="inference_pkg_ov" exec="inference_node" output="log">
    <param name="inference_engine" type="str" value="OV" />
  </node>
</launch>
